server/migrations/20240415225540_create_users_table.up.sql

```diff
@@ -0,0 +1,13 @@
+-- +migrate Up
+CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
```

Comment:
Never use serial keys pls. [UUID v7](https://pkg.go.dev/github.com/google/uuid#NewV7) are 128-bit long, collisions resistant, time-sortable and very fast to encode to strings for visual representation